libgeopmd/Makefile.am

```diff
@@ -348,13 +346,8 @@ io.github.geopm.xml:
 	fi
 
 # RPM TARGET
-rpm_topdir ?= $(HOME)/rpmbuild
-rpm: dist
-	mkdir -p $(rpm_topdir)/SOURCES
-	mkdir -p $(rpm_topdir)/SPECS
-	cp geopm-service-$(VERSION).tar.gz $(rpm_topdir)/SOURCES
-	cp geopm-service.spec $(rpm_topdir)/SPECS
-	rpmbuild $(rpmbuild_flags) -ba $(rpm_topdir)/SPECS/geopm-service.spec
+rpm:
+	./make_rpm.sh
```

Comment:
Double check that defining rpmbuild_flags with make rpm passes the flags through properly for things like levelzero enablement, etc.

A deprecation warning would also be nice.